pkg/didcomm/protocol/issuecredential/service.go

```diff
@@ -304,6 +327,21 @@ func (s *Service) handle(md *metaData) error {
 	return nil
 }
 
+func (s *Service) saveCredentials(md *metaData) error {
+	for i, credential := range md.credentials {
+		var name = credential.ID
+		if len(md.credentialNames) > i {
+			name = md.credentialNames[i]
+		}
+
+		if err := s.verifiable.SaveCredential(name, credential); err != nil {
```

Comment:
Any reason for inline err style here?

Reply:
No, I just prefer to use inline. Should not I?